build.rs

```diff
@@ -0,0 +1,73 @@
+
+extern crate build_const;
+
+pub fn make_table_crc32(poly: u32) -> [u32; 256] {
```

Comment:
does it make sense to move these two make table fns to a util file/module so that we don't have to repeat them in crc32.rs and crc64.rs